theme.json

```diff
@@ -88,11 +88,21 @@
           "radius": false
         }
       },
+      "core/heading": {
+        "spacing": {
+          "margin": true
+        }
+      },
       "core/group": {
         "spacing": {
           "padding": true
         }
       },
+      "p4/issues": {
```

Comment:
This is needed by https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/914